receiver/mongodbreceiver/README.md

```diff
@@ -0,0 +1,56 @@
+# Mongodb Receiver
+
+This receiver fetches stats from a Mongodb instance using the [golang
+mongo driver](https://github.com/mongodb/mongo-go-driver). Stats are collected
+via MongoDB's `dbStats` command.
+
+Supported pipeline types: `metrics`
+
+> :construction: This receiver is in **BETA**. Configuration fields and metric data model are subject to change.
+
+## Prerequisites
+
+This receiver supports MongoDB versions:
+
+- 2.6
+- 3.0+
+- 4.0+
+- 5.0
+
+## Configuration
+
+The following settings are required:
+
+- `hosts` (default: [`localhost:27017`]): list of `host:port` or unix domain socket endpoints.
+  - For standalone MongoDB deployments this is the hostname and port of the mongod instance
+  - For replica sets specify the hostnames and ports of the mongod instances that are in the replica set configuration. Another requirement is that the `replica_set` field must also be set with the name of the replica set for autodiscovery to work.
```

Comment:
If `replica_set` is specified, does the user only need to specify one mongod, or is it necessary to specify all of them?

Reply:
From my understanding, it is expected to specify the mongod instances which are specified in the configuration of the replica set. 

https://docs.mongodb.com/manual/reference/connection-string/#replica-set-option